src/Xamarin.Android.Build.Tasks/Tasks/ResolveSdksTask.cs

```diff
@@ -476,18 +476,16 @@ bool GetAapt2Version ()
 				);
 			} catch (Exception ex) {
 				Log.LogWarningFromException (ex);
-				Log.LogCodedWarning ("XA0034", $"Failed to get the Aapt2 version. Disabling Aapt2 Support.");
```

Comment:
Why remove this warning? It seems sensible. If anything, it should *precede* the `Log.LogWarningFromException(ex)`.

Reply:
`XA0034` is:
```
An exception was raised when Xamarin.Android tried to get the Java SDK version.
Make sure you have a valid java installation installed. 
```

Since we are returning `false` here, it will emit this warning:
```
Log.LogCodedWarning ("XA0111", "Could not get the `aapt2` version. Disabling `aapt2` support. Please check it is installed correctly.");
```